bot/exts/evergreen/emoji_count.py

```diff
@@ -0,0 +1,88 @@
+import datetime
+import logging
+import random
+from typing import Dict, Optional
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+log = logging.getLogger(__name__)
+
+
+class EmojiCount(commands.Cog):
+    """Command that give random emoji based on category."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    def embed_builder(self, emoji: dict) -> discord.Embed:
+        """Genrates embed with emoji name and count."""
+        embed = discord.Embed()
+        embed.color = Colours.orange
+        embed.title = "Emoji Count"
+        embed.timestamp = datetime.datetime.utcnow()
+        if len(emoji) == 1:
+            for key, value in emoji.items():
+                print(key)
+                embed.description = f"There are **{len(value)}** emojis in the **{key}** category"
+                embed.set_thumbnail(url=random.choice(value).url)
+        else:
+            msg = ''
+            for key, value in emoji.items():
+                emoji_choice = random.choice(value)
+                error_msg = f'There are **{len(value)}** emojis in the **{key}** category\n'
+                msg += f'<:{emoji_choice.name}:{emoji_choice.id}> {error_msg}'
+            embed.description = msg
+        log.trace('Emoji embed sent')
```

Comment:
It doesn’t get send here, something like `'Emoji embed built'` would make more sense, or move it where it is actually sent.